include/api.php

```diff
@@ -664,18 +664,20 @@ function api_item_get_user(&$a, $item) {
 
 		// Comments in threads may appear as wall-to-wall postings.
 		// So only take the owner at the top posting.
-		if ($item["id"] == $item["parent"])
-			$status_user = api_get_user($a,$item["owner-link"]);
-		else
-			$status_user = api_get_user($a,$item["author-link"]);
-
+		#if ($item["id"] == $item["parent"])
```

Comment:
I don't really like this kind of commenting out stuff. What is the benefit of not using "//"?